CentralHub.Api/Controllers/MeasurementController.cs

```diff
@@ -87,7 +87,29 @@ public async Task<GetAggregatedMeasurementsResponse> GetAggregateMeasurements(in
         return GetFirstAggregatedMeasurementsDateTimeResponse.CreateSuccessful(possibleFirstDateTime.Value);
     }
 
-    private static IReadOnlyList<AggregatedMeasurements> CreateMeasurements(IReadOnlyCollection<AggregatedMeasurementDto> aggregatedMeasurements)
+    [HttpGet("occupancy/latest")]
+    public async Task<GetLatestOccupancyResponse> GetLatestEstimatedOccupancy(
+        int roomId,
+        float wifiDevicesPerPerson,
+        float bluetoothDevicesPerPerson,
+        CancellationToken cancellationToken)
+    {
+        var aggregatedMeasurements = (await measurementRepository.GetAggregatedMeasurementsAsync(roomId, cancellationToken)).Last();
+
+        if (aggregatedMeasurements == null)
+        {
+            return GetLatestOccupancyResponse.CreateUnsuccessful();
+        }
+
+        var occupancy =
+            (int)Math.Round(
+            bluetoothDevicesPerPerson == 0 ? 0 : aggregatedMeasurements.BluetoothCount / bluetoothDevicesPerPerson +
+            wifiDevicesPerPerson == 0 ? 0 : aggregatedMeasurements.BluetoothCount / wifiDevicesPerPerson);
```

Comment:
integer division? but i guess that is okay in this case, however, it will always floor

Reply:
pretty surte its floating point devision, so its fine

Reply:
Its not floating point division